next.config.js

```diff
@@ -104,10 +104,5 @@ module.exports = withSwingset({
 			dangerouslyAllowSVG: true,
 			contentSecurityPolicy: "default-src 'self'; script-src 'none'; sandbox;",
 		},
-		experimental: {
-			images: {
-				layoutRaw: true,
-			},
-		},
```

Comment:
No longer required for next `12.3.0` correct?

Reply:
Correct, also now that `next/future/image` is stable!